Client/mods/deathmatch/logic/luadefs/CLuaBuildingDefs.cpp

```diff
@@ -30,10 +30,10 @@ void CLuaBuildingDefs::AddClass(lua_State* luaVM)
 
     lua_classfunction(luaVM, "create", "createBuilding");
 
-    lua_registerclass(luaVM, "Building");
+    lua_registerclass(luaVM, "Building", "Element");
 }
 
-CClientBuilding* CLuaBuildingDefs::CreateBuilding(lua_State* const luaVM, uint16_t modelId, CVector pos, CVector rot, std::optional<uint8_t> interior)
+CClientBuilding* CLuaBuildingDefs::CreateBuilding(lua_State* const luaVM, uint16_t modelId, CVector pos, std::optional<CVector> rot, std::optional<uint8_t> interior)
```

Comment:
According to the encoding guidelines, std::uint16_t, std::uint8_t, etc. should be used.